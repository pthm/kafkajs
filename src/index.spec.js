const Client = require('./index')

describe('Client', () => {
  it('gives access to its logger', () => {
    expect(new Client({ brokers: [] }).logger()).toMatchSnapshot()
  })
})
