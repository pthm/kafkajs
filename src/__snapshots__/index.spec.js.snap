// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Client gives access to its logger 1`] = `
Object {
  "debug": [Function],
  "error": [Function],
  "info": [Function],
  "namespace": [Function],
  "warn": [Function],
}
`;
